src/sage/combinat/tutorial.py

```diff
@@ -844,15 +844,15 @@
 
 Partial orders on a set of `8` elements, up to isomorphism::
 
-    sage: C = Posets(8); C
-    Posets containing 8 elements
+    sage: C = Posets(7); C
+    Posets containing 7 elements
     sage: C.cardinality()
-    16999
+    2045
```

Comment:
well, I think smaller doctests are always better. No need to show muscles in doctests.

Reply:
OK, I don't mind - although I somewhat disagree, with two (implicit) statements:

* in the case at hand, we are only looking up the number
* in general, I think it it good to use the largest example which takes a reasonable amount of time (in particular, less than a second), because it seems to me that this helps to catch bad performance regressions.